pygfx/renderers/wgpu/_renderer.py

```diff
@@ -115,7 +113,6 @@ def __init__(
         blend_mode="default",
         sort_objects=False,
         enable_events=True,
-        prefer_srgb=False,
```

Comment:
IMHO,  The original way may be better (use `prefer_srgb` flag)

Reply:
The target textures are now always sRGB, there is no way to turn it off anymore.

Reply:
The `prefer_srgb` was an attempt to fix the issue before I understood well enough how all this works :)